### PR TITLE
Quick intro for php

### PR DESCRIPTION
**Pull request by StevenKW**
A quick intro for the PHP section.

### CODE REVIEW

File: docs/ABOUT.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#Introduction
```
Comment (kytrinyx) [inline docs/ABOUT.md]: This is going to be shown on a page that already has a header, so this line should be deleted.

File: docs/ABOUT.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Fast, flexible and pragmatic, PHP powers everything from your blog to the most popular websites in the world.
+
+What distinguishes PHP from something like client-side JavaScript is that the code is executed on the server, generating HTML which is then sent to the client. 
+ 
+PHP is extremely simple for a newcomer, but offers many advanced features for a professional programmer.
+
+##What can PHP do?
+Anything. PHP is mainly focused on server-side scripting, so you can do anything any other CGI program can do, such as collect form data, generate dynamic page content, or send and receive cookies. But PHP can do much more.
+
+More PHP details can be found  [here](http://php.net/)
```
Comment (kytrinyx) [inline docs/ABOUT.md]: This section feels a bit distracting--PHP is not usually used to write satellite software or program toaster ovens.

File: docs/ABOUT.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+##Why should you learn PHP?
+According to W3 Techs PHP accounts for a huge percent of server-side programming languages for websites. There should be plenty of demand for PHP programmers now and in the future.
```
Comment (kytrinyx) [inline docs/ABOUT.md]: The reader might not know (or care, or know why they _should_ care) who W3 Techs are.

It's not entirely clear to me what _accounts for a huge percent of server-side programming languages for websites_ means. PHP is only one language, so it's not a huge percent of the programming languages. Do you mean that a huge percent of websites are written in PHP?

**Comment by petemcfarlane** [comment]
Closing to clean up the PRs, this seems quite out of date. Feel free to reopen if I'm wrong!

**Pull request closed without merging**